contracts/Example3.sol

```diff
@@ -1,14 +1,15 @@
+
+// SPDX-License-Identifier: MIT
 pragma solidity ^0.8.4;
 
 contract Example3 {
-    uint256 marginPercentage = 30;
     function getOwnerMargin(uint256 amount)
         public
         view
         returns (uint256 amountForSender, uint256 amountForOwner)
     {
-        amountForSender = (amount * (100 - marginPercentage)) / 100;
-        amountForOwner = (amount * marginPercentage) / 100;
+        amountForSender = (amount * 70) / 100;
```

Comment:
It is best to use the saved variable rather that hardcoding the values here. The catch here was to store the `marginPercentage` in memory since it is being accessed twice